apps/projects/app/components/Content/IssueDetail/Label.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { textStyle, unselectable, useTheme } from '@aragon/ui'
+
+const Label = ({ text, block, children }) => {
+  const theme = useTheme()
+  return (
+    <label
+      css={`
+        ${textStyle('label2')};
+        ${unselectable()};
+        color: ${theme.surfaceContentSecondary};
+        display: ${block ? 'block' : 'initial'};
+      `}
+    >
+      {text}
+      {children && children}
```

Comment:
There was a case long time ago when it needed to be checked - the difference I think (I might be wrong though) is that `{children}` will be passed further down even if it is undefined or null, and `{children && children}` will not. And then if the target component tries to do something with that prop, it will fail. That was happening in FilterBar when... DropDown was calling focus() (or something like that) on undefined children. But in this case that extra check is not needed, so I removed it.